acceptance/bundle/resources/pipelines/recreate/_script

```diff
@@ -0,0 +1,36 @@
+trace cat databricks.yml
+touch foo.py
+touch bar.py
+trace $CLI bundle plan  # should show 'create'
```

Comment:
Can it be part of asserting that it should be `create`?

Reply:
You mean something like assert plan == "create pipelines.my"? 

We could have a wrapper script for plan that asserts, but I think having the output recorded is good enough as well.

Reply:
It seems like instead of having a comment here it would be best to do the `grep "create pipelines.my"`, but that's up to you

Reply:
but what if plan shows more than that? That would be a bug but grep would hide that. Also, even if plan is wrong, it for debugging it would be useful to ahead with deploy to see more output.